exercises/hello-workflow/README.md

```diff
@@ -0,0 +1,74 @@
+# Exercise 1: Hello Workflow
+
+During this exercise, you will
+* Review the business logic of the provided Workflow Definition to understand its behavior
+* Modify the Worker initialization code to specify a task queue name
+* Run the Worker initialization code to start the Worker process
+* Execute the Workflow from the command line, specifying your name as input
+
+Make your changes to the code in the `practice` subdirectory (look for TODO 
+comments that will guide you to where you should make changes to the code). 
+If you need a hint or want to verify your changes, look at the complete version 
+in the `solution` subdirectory.
+
+## Part A: Review the Workflow Business Logic
+
+1. Open the `Workflow.cs` file (located in the `Worker` folder under `practice`) 
+   in the editor
+2. Review the input parameters, business logic, and return value. 
+
+## Part B: Specify a Task Queue Name for the Worker
+
+1. Open the `Program.cs` file (located in the `Client` folder under `practice`) 
+   in the editor
+2. Write `greeting-tasks` as the name of the task queue
+3. Save your changes
+4. Repeat and open the `Program.cs` file in the `Worker` folder under `practice)
+5. Write `greeting-tasks` as the name of the task queue and save
+
+
+## Part C: Start the Worker
+
+1. Make sure you're in the practice subdirectory. Run the following command in the terminal window to start the Worker
+
+```
+dotnet run --project Worker
+```
+
+Note: If you're running this on your local, make sure you are running the Temporal Service. In a new terminal window, run the following command: `temporal server start-dev`
+
+## Part D: Start the Workflow from the Command Line
+
+1. Open another terminal window in the environment and change to the 
+   `Client` folder under `practice` for this exercise 
+2. Run the following command, replacing `Donna` with your own name. 
+   Be sure to retain the same quoting shown here when you run the command:
+
+```bash
```

Comment:
Not sure if we want to remove the `\` linebreaks and put this all on one line just for the dotnet course because I imagine we will have a lot of Windows users and I don't think these work in Powershell. I personally use bash on Windows, but I don't think enough people do.